src/components/join/How.jsx

```diff
@@ -1,9 +1,61 @@
-const How = () => {
+const MembershipInstructions = () => {
   return (
-    <div className="">
-      <p>How</p>
+    <div className="max-w-4xl mx-auto p-6 bg-white shadow rounded-lg">
+      <h1 className="text-3xl font-bold mb-4">
```

Comment:
stick with divider tags instead of header tags. header tags might mess up the font size when we deploy the website

Reply:
the title color should be blue (use the projects color palette, not tailwind's default colors).